lib/vyasa_web/components/source_content/chapters.ex

```diff
@@ -0,0 +1,73 @@
+defmodule VyasaWeb.Content.Chapters do
```

Comment:
intent: as much as possible, all the live_components should handle events local to them (e.g. this navigation event). This is why the `@myself` for the event targets have been used. 

note that without explicit definition, the parent live_view module is the one that shall handle events triggerred from child live_components.